### PR TITLE
Stojanovic/fe 85 hide published in text when no publisher in metadata

### PR DESCRIPTION
**Pull request by Ducica**
@mirekys fixes per FE-85, FE-90, FE-93, FE-94, FE-95, FE-96. Mainly aesthetical changes and some translations. Please take a look.

### CODE REVIEW

File: sites/nr-docs/pyproject.toml [hunk 1]
```diff
@@ -11,6 +11,8 @@ dependencies = [
     "nr-documents @ file:///${PROJECT_ROOT}/../../models/nr-documents",
     "docs-app @ file:///${PROJECT_ROOT}/../../ui/docs-app",
     "nr-docs-extensions @ file:///${PROJECT_ROOT}/../../local/nr-docs-extensions",
+    "oarepo-ui @ file:///${PROJECT_ROOT}/../../local/oarepo-ui",
```
Comment (mirekys) [inline sites/nr-docs/pyproject.toml:14]: Remove this 2 local deps

File: sites/nr-docs/requirements.txt [hunk 2]
```diff
@@ -13,9 +13,7 @@ nr-vocabularies==2.0.5
 oarepo-oai-pmh-harvester==4.0.28
 oarepo-published-service==1.0.1
 oarepo-requests==1.0.2
-oarepo-runtime==1.4.9
```
Comment (mirekys) [inline sites/nr-docs/requirements.txt:16]: Don't remove ui + vocab deps

File: ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx [hunk 3]
```diff
@@ -239,9 +241,7 @@ export const DepositForm = () => {
                       placeholder={i18next.t("Choose licenses")}
                       clearable
                       optionsListName="licenses"
-                      helpText={i18next.t(
-                        "If a Creative Commons license is associated with the resource, select the appropriate license option from the menu. We recommend choosing the latest versions, namely 3.0 Czech and 4.0 International."
-                      )}
+                      helpText={i18next.t("Creative Commons license")}
```
Comment (mirekys) [inline ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx:244]: Why this change? There are possibly multiple licences other than CC

File: ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/search/components/ResultsItemAccessStatus.jsx [hunk 4]
```diff
@@ -8,7 +8,7 @@ export const ResultsItemAccessStatus = ({ status }) => {
     iconFile && (
       <Image
         as="a"
-        href={`/vocabularies/${type}/${id}`}
```
Comment (mirekys) [inline ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/search/components/ResultsItemAccessStatus.jsx:11]: Why not use type from props? is it empty & why? If it couldn't be passed via type prop, remove the prop as it will be unused

Comment (Ducica) [reply ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/search/components/ResultsItemAccessStatus.jsx:11]: @mirekys Zdar, the vocabulary info in metadata looks like this

image

So only the basic data for showing it and no type included. As the component is related specifically to access-rights vocabulary, I thought hardcoding it would not be such a problem. I removed the type from props.

File: ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx (hunk 3 above)
Comment (Ducica) [reply ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx:244]: @mirekys I only changed the key, as I had some weird issue with key not being found even though it exists in translation.json, so I thought maybe the key was too long or something :) and it worked, but now when I returned the key as full string "If a Creative Commons license is associated with the dataset, select the appropriate license option from the menu. We recommend choosing the latest versions, namely 3.0 Czech and 4.0 International.", it seems to be working also, so I returned it like that. Sorry about that..

File: ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx [hunk 5]
```diff
@@ -240,7 +242,7 @@ export const DepositForm = () => {
                       clearable
                       optionsListName="licenses"
                       helpText={i18next.t(
-                        "If a Creative Commons license is associated with the resource, select the appropriate license option from the menu. We recommend choosing the latest versions, namely 3.0 Czech and 4.0 International."
+                        "If a Creative Commons license is associated with the dataset, select the appropriate license option from the menu. We recommend choosing the latest versions, namely 3.0 Czech and 4.0 International."
```
Comment (mirekys) [inline ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx:245]: Publications & documents != datasets :)

**Review by mirekys — APPROVED** [review]

**Review by mirekys — APPROVED** [review]

**Review by mirekys — APPROVED** [review]
